### PR TITLE
Create font-skola-sans.rb

### PR DESCRIPTION
**Pull request by smargh**
This is a lovely and large sans serif font family on Bechance. I use it on my personal blog and wanted to have an easy way to download it on new machines.

### CODE REVIEW

File: Casks/font-skola-sans.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+cask :v1 => 'font-skola-sans' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://download1001.mediafire.com/b61raazyg9cg/5cbb1kb41ypzm3v/SkolaSansPro.zip'
```
Comment (vitorgalvao) [inline Casks/font-skola-sans.rb]: Needs a `# mediafire.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

**Comment by smargh** [comment]
Didn't know that. Thanks for the heads up. Committed the change.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by smargh** [comment]
Squashed

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**